src/config/dynamic-theme-fixes.config

```diff
@@ -8267,6 +8236,46 @@ INVERT
 
 ================================
 
+whatsapp.com
+
+INVERT
+.selectable-text > span._3PxOr
+span[data-icon="audio-download"]
+.landing-main .invisible-space > span
+.landing-main div[data-ref] span > svg > path
+div.landing-header > span > svg
+#wafaq_search_input
+
+CSS
+._3Ye_R {
```

Comment:
Not sure when this was introduced, but can confirm it's not more in use and can be removed.